index.md

```diff
@@ -53,6 +53,10 @@ PING occasionally uses IRC on the [#privacy](http://irc.w3.org/?channels=privacy
 * Port: 6667 or 6665
 * Channel: #privacy
 
+## Documentation and Guidance
+- [Mitigating Browser Fingerprinting in Web Specifications](https://w3c.github.io/fingerprinting-guidance/)
```

Comment:
This puts two links to that doc on the same page.  How can we arrange things so that there isn't the redundancy?

Reply:
How does this look?  I merged the two link sections (and fixed some typos and capitalization consistency issues)